java/src/main/native/src/AggregationJni.cpp

```diff
@@ -120,10 +114,10 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_Aggregation_createDdofAgg(JNIEnv *en
     std::unique_ptr<cudf::aggregation> ret;
     // These numbers come from Aggregation.java and must stay in sync
     switch (kind) {
-      case 9: // VARIANCE
+      case 10: // VARIANCE
```

Comment:
imo, best practice is to not change ids unless absolutely necessary.

Reply:
I agree. Unfortunately, the new aggregations are not added to the end of the list but in the middle.
I am trying to find some better way to avoid adding those enum values manually, will eventually come up with something better so we will not have to worry about this issue anymore.

Reply:
Note that the enum you're updating here does _not_ need to match the cudf version.  It only needs to be self-consistent between Aggregation.java and AggregationJni.cpp.  You could add the new values to the end of that enumeration and significantly reduce the size of this patch.

Reply:
Thanks Jason. I'm aware of that. But again, this is not a long term solution, and I want to find a better way for this. But yeah, I will just add the new aggregations to the end of the list for better readability now. I will come up with a better solution in the future which can totally solve this.